ge10_team01/static/src/snippets/s_registry_miles/000.js

```diff
@@ -0,0 +1,100 @@
+odoo.define('ge10_team01.s_registry_miles', function (require) {
```

Comment:
Why did you name the file like this?

Reply:
To be honest just because I saw that's how the file was named almost all the other snippets. Also in the task hints it's told the file that is in charge of the UI is named 000.js.